### PR TITLE
[clang][cas] Disable the `#include_next` machinery while replaying an include-tree

### PR DESCRIPTION
**Pull request by akyrtzi**

### CODE REVIEW

**Comment by akyrtzi** [comment]
@swift-ci Please test

File: clang/test/CAS/include-tree-with-include-next.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+// RUN: %clang -cc1depscan -o %t/tu.rsp -fdepscan=inline -fdepscan-include-tree -cc1-args \
+// RUN:     -cc1 -fcas-path %t/cas %t/t.c -I %t/inc -I %t/inc2 -Werror
+// RUN: %clang @%t/tu.rsp -fsyntax-only -Werror
+
+//--- t.c
+#include "t.h"
+
+//--- inc/t.h
+#include_next "t2.h"
```
Comment (cachemeifyoucan) [inline clang/test/CAS/include-tree-with-include-next.c]: Shouldn't the include_next test usually test include the next header with the same name?

**Review by cachemeifyoucan — COMMENTED** [review]
LGTM

File: clang/test/CAS/include-tree-with-include-next.c (hunk 1 above)
Comment (akyrtzi) [reply clang/test/CAS/include-tree-with-include-next.c]: Good catch, technically it doesn't matter for this test but it does look strange, I'll update.

**Pull request merged**